src/components/reservations/ReservationReleasePDF.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Button, IconDocument, IconDownload } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+
+import { useGetCustomerByIdQuery } from '../../redux/services/api';
+import { useDownloadFile } from '../../utils/useDownloadFile';
+import { useFileDownloadApi } from '../../utils/useFileDownloadApi';
+import { toast } from '../common/toast/ToastManager';
+
+import styles from './ReservationReleasePDF.module.scss';
+
+const T_PATH = 'components.reservations.ReservationReleasePDF';
+
+interface Props {
+  reservationId: number;
+  customerId: number;
+  size?: 'small' | 'normal';
```

Comment:
Is this actually relevant? I cannot see this being referenced outside of this component.